crates/swc_ecma_utils/src/lib.rs

```diff
@@ -2520,11 +2520,6 @@ impl ExprCtx {
                 to.push(Box::new(Expr::New(e)))
             }
             Expr::Member(_) | Expr::SuperProp(_) => to.push(Box::new(expr)),
-            Expr::OptChain(OptChainExpr { ref base, .. })
```

Comment:
This is obviously wrong -- hasn't considered computed keys, and I'm worried about nested case too, so it's economical to just remove it.